### ISSUE TITLE
Advanced search form

### ISSUE THREAD

**Issue by adehner** [scholarslab/SolrSearch#128]
Thank you for creating this plugin. The faceting and indexing options greatly improve searching in Omeka.

However, we miss advanced search fields. Is this an option that you are considering for a future release? And in the meantime, would you be willing to point me in the direction of creating and advanced search form for my installation?

Thanks for considering!

**Comment by adehner** [comment]
Hi there,

I've been digging in to documentation to add advanced search capability to my Omeka Solr Search instance. I see that the fields used by the plugin are dynamic fields, including the element-text-id. However when I try to construct a query with these fields, my results are exactly the same as a general keyword search.

Is there something I can do to search within one of these dynamic fields?

Any tips you have are much appreciated!!

**Comment by adehner** [comment]
Just following up on what I discovered, to aid future inquirers...

The plugin's controller specifically filters out colons in the search term, at least in the "q" parameter: [https://github.com/scholarslab/SolrSearch/blob/master/controllers/ResultsController.php#L119](https://github.com/scholarslab/SolrSearch/blob/master/controllers/ResultsController.php#L119)

To create a solr advanced search with this plugin, I first commented out the line above. Then I was able to construct pre-filtered query strings (eg. /solr-search?q=63_s:Frankfort+AND+62_s:1864). I didn't notice any negative consequences from commenting out line 119, however the plugin authors might be able to speak to the potential problems of this change.
